erpnext/hr/doctype/training_event/training_event.js

```diff
@@ -17,5 +17,37 @@ frappe.ui.form.on('Training Event', {
 				frappe.set_route("List", "Training Feedback");
 			});
 		}
+	},
+	onload: function(frm) {
+		var params = getSearchParameters();
+		if (params.hasOwnProperty('employee') && params.hasOwnProperty('status')) {
+			var newTemp = frm.doc.employees.filter(function(obj) {
+				return obj.name == params.employee;
+			});
+			if (newTemp) {
+				newTemp[0].status = params.status;
+				frm.refresh_field("employees");
+				frappe.msgprint(__(frm.doc.name + ": Status for " + newTemp[0].employee_name +
```

Comment:
`frappe.msgprint(__('{0}: Status for {1} is updated to {2}', [arg0, arg1, arg2]))` works better for translated messages